test/include/autoinstall/AutoinstallAskInclude_test.rb

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env rspec
```

Comment:
interesting, in rails word if I remember it was unit/ functional etc and it do not copy structure of app or lib. So for me using include make sense, using module namespace there, not much sense.